core/src/main/java/org/apache/iceberg/encryption/EncryptionManagers.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.encryption;
+
+public class EncryptionManagers {
+  private static final EncryptionManager DEFAULT_MANAGER = new PlaintextEncryptionManager();
```

Comment:
Maybe we call it PLAIN_TEXT_MANAGER? so that reader can get it right away instead of checking class `PlaintextEncryptionManager`. Besides, this is not introduced by this patch, but the name of `PlaintextEncryptionManager` itself is also confusing. If we cannot change it to a more descriptive name, we may add a comment in the class to indicate it doesn't do any encryption/decryption.